torch_glow/tests/nodes/quantized_conv2d_test.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+import pytest
+
+
+def test_quantized_conv2d():
+    """Basic test of the PyTorch quantized::relu Node on Glow."""
+
+    def test_f(a, w, b):
+        qu = torch.nn.quantized.Quantize(1/16, 0, torch.quint8)
+        qi = torch.nn.quantized.Quantize(1/16, 0, torch.qint8)
+        dq = torch.nn.quantized.DeQuantize()
+        conv = torch.nn.quantized.functional.conv2d
+        return dq(conv(qu(a), qi(w), b))
+
+    # TODO
+    # Due to the quantization error between
+    # PyTorch and Glow, we would like to use some
+    # determined test data instead of random ones
+    # x = torch.randn([3, 3, 3, 3])
+    # w = torch.randn([3, 3, 3, 3])
+    # b = torch.randn([3])
+
+    x = torch.tensor([[[[5., 6.], [7., 8.]]]])
+    w = torch.tensor([[[[1., 2.], [3., 4.]]]])
+    b_zero = torch.zeros(1)
+    b = torch.randn(1)
+
+    jitVsGlow(test_f, x, w, b, expected_fused_ops={"aten::quantize_per_tensor",
+                                                   "glow::unpacked_quantized_conv2d",
+                                                   "aten::dequantize"})
+
+    jitVsGlow(test_f, x, w, b_zero, expected_fused_ops={"aten::quantize_per_tensor",
+                                                        "glow::unpacked_quantized_conv2d",
+                                                        "aten::dequantize"})
+
+
+@pytest.mark.skip(reason="accuracy between glow&ytorch")
```

Comment:
I'm not sure I understand, it looks the same but just has random weight initialization? Won't this always be different accross different runs?

Reply:
Yes, but,
1. it has a empty_affine_quantize out of our fusion node, which gives a quantized tensor as a fusion node input, and invoke the code we have in https://github.com/pytorch/glow/pull/3635. Without 3635 it will crush.

2. Since it has random weight initialization, it generate error, and have a great chance to fail, although sometimes it can pass. This flakiness is similar to why we cannot have random float number test in regular quantized conv.

Reply:
2. Yeah exactly, if it's random initializing weights, we can't run it twice and compare the results so it's probably not a good test

Reply:
> 1. Yeah exactly, if it's random initializing weights, we can't run it twice and compare the results so it's probably not a good test

Agree, however that is the only way I know to exclude a quantized tensor from our fusion node, or else it will be fused in...

Reply:
Why can't we provide a hand written set of weights?

Reply:
> Why can't we provide a hand written set of weights?

I can try and enbale this test in that new diff.